src/registrar/admin.py

```diff
@@ -1868,6 +1885,28 @@ def status_history(self, obj):
 
     change_form_template = "django/admin/domain_request_change_form.html"
 
+    # While the organization feature is under development, we can gate some fields
+    # from analysts for now. Remove this array and the get_fieldset overrides once this is done.
+    # Not my code initially, credit to Nicolle. This was once removed and like a phoenix it has been reborn.
+    superuser_only_fields = [
```

Comment:
I did not catch this on the first pass but I think we should simply have these fields as readonly for analysts. This AC indicates so:

_The Admin Readonly view for Portfolio Request, Domain info/Domain does not show the requested suborganization, suborg_city and state fields if the user is selecting a suborganization from the list. But it displays the requested suborganization, city/state fields when the user selects the "Other" option._

If that's the case, we should use pre-established patters for analyst readonly / superuser editable fields.

Reply:
I would agree read only better follows our pre-existing patterns. However, I do think design wants those fields to not show. That said, @Katherine-Osos could we have the requested suborganization, suborg_city and state fields just turn read-only when suborganization is filled in and still provide a similiar user experience for analysts?